docs/configure_admin_workstation_post_install.rst

```diff
@@ -91,25 +61,11 @@ The install scripts installs a Network Manager hook that runs on boot. It automa
 SSH Host Aliases
 ----------------
 
-.. note:: This step is optional, but it makes it much easier to
-	  connect to the servers in order to administrate them in the future.
-
-Create the file ``/home/amnesia/.ssh/config`` and add a configuration
-following the scheme below, replacing ``Hostname`` and ``User`` with
-the values specific to your setup:
-
-::
-
-    Host app
-      Hostname <app hostname>.onion
-      User <ssh username>
-    Host mon
-      Hostname <mon hostname>.onion
-      User <ssh username>
+The installation script in ``tails_files`` also sets up SSH host aliases for the admin.
```

Comment:
So this is less "magic"-seeming, maybe add a note that you can see the host aliases added by the install script in `~/.ssh/config`.